src/MusicStore/project.json

```diff
@@ -5,9 +5,6 @@
     "description": "Music store application on ASP.NET 5",
     "version": "1.0.0-*",
     "compilationOptions": { "warningsAsErrors": true, "define": [ "DEMO", "TESTING" ] },
-    "compile": [
-        "../../test/E2ETests/compiler/shared/**/*.cs"
-    ],
```

Comment:
I am not sure the effects of removing these(like tests other than the ones testing 'publish' itself)...my idea was that if it only effecting publish tests, then we could disable them specifically...did you try running build.cmd and see if this has any effect?